### DISCUSSION TITLE
optimized_img2img.py issue

### DISCUSSION THREAD

**Discussion by shade1er** [basujindal/stable-diffusion#45] [Q&A]
in module init_latent ... object has no attribute '{}'".format( AttributeError: 'FirstStage' object has no attribute 'get_first_stage_encoding'

DAMN IT :)
any ideas ? thanks !

**Comment by GucciFlipFlops1917** [comment]
What prompt are you feeding in? Also, ideal to post this under "Issues".

**Reply by shade1er** [reply]
Hi,
nothing special "beautiful landscape". 
input image is 512x512px, command line is a copy paste of example here with correct W and H

**Reply by GucciFlipFlops1917** [reply]
I think your issue is similar to the one here:
https://github.com/basujindal/stable-diffusion/issues/16

If it is what @oxysoft is describing, you may have luck by adding this line after where `import sys` occurs:
`sys.path.append('.')` 

The generation scripts were also updated yesterday. It at least fixed the img2img file for me, so that's a possibility if you downloaded a slightly older version.

**Reply by shade1er** [reply]
oh thanks, i'll update it right now to see if it's fixed.
i've cloned the git 2 days ago i think !

**Reply by shade1er** [reply]
same error.
